deploy/vmscrape.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     matchExpressions:
       - key: name
         operator: In
-        values: [autoscaler-agent, autoscale-scheduler]
+        values: [autoscaler-agent, autoscale-scheduler, neonvm-controller]
   namespaceSelector:
     matchNames:
       - kube-system
```

Comment:
IIUC, we can't simply add `neonvm-controller` here for a couple reasons:

1. This only matches pods with the label `name=neonvm-controller`; the controller doesn't have that label
2. This only matches pods in the `kube-system` namespace (`neonvm-controller` is in `neonvm-system`).

Maybe have a separate `VMPodScrape` below this for `neonvm-system` that uses a different label?

Reply:
fixed in 6ea7a9c04ca5e5662a0ec4ae5c3d6776ae939b84